pkg/server/server.go

```diff
@@ -206,6 +206,20 @@ func (s *Server) getLocalAuther() *auth.Authenticator {
 	return s.Authers[serverutils.LocalClusterName]
 }
 
+func disableDirectoryListing(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// If the request is for a directory, return a 404.
+		// Directory path is expected to end with a slash or be empty,
+		// since we are stripping the '/static/' prefix from the path.
+		if strings.HasSuffix(r.URL.Path, "/") || r.URL.Path == "" {
```

Comment:
Does this work if we try to visit `/static/assets` (no trailing slash)? Seems like this would cause the path to be `assets`, which wouldn't meet either of these conditions.

Reply:
It should be working since the `/static/assets` is redirected to `/static/assets/`. Will test.

Reply:
@TheRealJon confirming that user will get redirected to `/static/assets/` and 404 is shown.